syntax = "proto3";

message VtsAvFrame {
  int32 version = 1;
  repeated bytes rgbPackets = 2;
  repeated bytes aPackets = 3;
  int64 pts = 4;
}
